databricks/koalas/spark/accessors.py

```diff
@@ -993,6 +993,45 @@ def coalesce(self, num_partitions: int) -> "ks.DataFrame":
         coalesced_sdf = internal.spark_frame.coalesce(num_partitions)
         return DataFrame(internal.with_new_sdf(coalesced_sdf))
 
+    def checkpoint(self, eager=True):
+        """Returns a checkpointed version of this DataFrame.
+
+        Checkpointing can be used to truncate the logical plan of this DataFrame, which is
+        especially useful in iterative algorithms where the plan may grow exponentially. It will be
+        saved to files inside the checkpoint directory set with `SparkContext.setCheckpointDir`.
+
+        Parameters
+        ----------
+        eager : bool
+            Whether to checkpoint this DataFrame immediately
+
+        Returns
+        -------
+        DataFrame
+
+        .. note:: Experimental
+
+        Examples
+        --------
+        >>> kdf = ks.DataFrame({"a": ["a", "b", "c"]})
+        >>> kdf  # doctest: +NORMALIZE_WHITESPACE
```

Comment:
nit: I don't think we need `# doctest: +NORMALIZE_WHITESPACE`?

Reply:
Removed.